client/signup/steps/plans/index.jsx

```diff
@@ -162,6 +170,13 @@ export class PlansStep extends Component {
 		const fallbackHeaderText = this.props.fallbackHeaderText || headerText;
 		const subHeaderText = this.props.subHeaderText;
 
+		let backUrl = undefined,
+			backLabelText = null;
```

Comment:
Same question as [above](https://github.com/Automattic/wp-calypso/pull/31235/files#r263235406)

Reply:
Implemented in [cbb4f6f](https://github.com/Automattic/wp-calypso/pull/31235/commits/cbb4f6fa614e6d2a3c44b688d615cf630cbd5d6c).